test/unittests/test_extend_tt.cpp

```diff
@@ -16,11 +16,11 @@ TEST_F(TruthTableExtend, Max) {
     // create identity truth table
     std::string circMax = testCircuitsDir + "max.pla";
 
-    EXPECT_TRUE(readPla(tt, circMax));
+    EXPECT_ANY_THROW(readPla(tt, circMax));
```

Comment:
why is this test expected to throw? shouldn't parsing a regular `pla` file just work?

Reply:
The max.pla file has 64 inputs. As a result, the `extend(tt)` throws the overflow exception.